db/schema.rb

```diff
@@ -590,6 +607,42 @@
     t.index ["school_id", "school_group_id"], name: "index_school_group_memberships_on_school_id_and_school_group_id", unique: true
   end
 
+  create_table "self_disclosure_requests", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "job_application_id", null: false
+    t.integer "status"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["job_application_id"], name: "index_self_disclosure_requests_on_job_application_id", unique: true
+  end
+
+  create_table "self_disclosures", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "name_ciphertext"
+    t.string "previous_names_ciphertext"
+    t.string "address_line_1_ciphertext"
+    t.string "address_line_2_ciphertext"
+    t.string "city_ciphertext"
+    t.string "country_ciphertext"
+    t.string "postcode_ciphertext"
+    t.string "phone_number_ciphertext"
+    t.string "date_of_birth_ciphertext"
+    t.string "has_unspent_convictions_ciphertext"
+    t.string "has_spent_convictions_ciphertext"
+    t.string "is_barred_ciphertext"
+    t.string "has_been_referred_ciphertext"
+    t.string "is_known_to_children_services_ciphertext"
+    t.string "has_been_dismissed_ciphertext"
+    t.string "has_been_disciplined_ciphertext"
+    t.string "has_been_disciplined_by_regulatory_body_ciphertext"
+    t.string "agreed_for_processing_ciphertext"
+    t.string "agreed_for_criminal_record_ciphertext"
+    t.string "agreed_for_organisation_update_ciphertext"
+    t.string "agreed_for_information_sharing_ciphertext"
+    t.uuid "self_disclosure_request_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["self_disclosure_request_id"], name: "index_self_disclosures_on_self_disclosure_request_id", unique: true
+  end
+
```

Comment:
oops - you have committed the wrong schema.rb

Reply:
:recycle: